pkgs/native_assets_cli/test/validator/validator_test.dart

```diff
@@ -324,16 +315,15 @@ void main() {
     final output = BuildOutput();
     final assetFile = File.fromUri(outDirUri.resolve('foo.dylib'));
     await assetFile.writeAsBytes([1, 2, 3]);
-    output.addAsset(DataAsset(
+    output.dataAssets.add(DataAsset(
       package: 'different_package',
       name: 'foo.txt',
       file: assetFile.uri,
     ));
-    final result = await validateBuild(config, output);
-    expect(result.success, isFalse);
+    final errors = await validateDataAssetBuildOutput(config, output);
```

Comment:
Changing the test in this way shows that default validation no longer validates asset IDs.